todo-app/src/components/Todos/ListTodos.js

```diff
@@ -1,43 +1,47 @@
 // jshint esnext: true
 
 import React from 'react';
-import TodoItem from './Todo-item';
+import TodoItem from './TodoItem';
+import './styles/ListTodos.css';
 
 
 let todosInfo = [
     {
         id: 1,
         author: 'person 1',
-        date: new Date().toLocaleString(),
+        date: new Date(),
         avatar: 'https://d30y9cdsu7xlg0.cloudfront.net/png/770852-200.png',
+        alt: this.author + ' image',
```

Comment:
If your logic for the `alt` attribute is simply `todo.author + ' image'`, you can do that directly inside the component and save repetition here.